lib/react_client.dart

```diff
@@ -1020,6 +1157,25 @@ dynamic _findDomNode(component) {
   return ReactDom.findDOMNode(component is Component ? component.jsThis : component);
 }
 
+class ReactDartContext {
+  ReactDartContext(this.Provider, this.Consumer, this._jsThis);
+  final ReactContext _jsThis;
+  final ReactJsContextComponentFactoryProxy Provider;
+  final ReactJsContextComponentFactoryProxy Consumer;
+  ReactContext get jsThis => _jsThis;
+}
+
+ReactDartContext createContext([
+  dynamic defaultValue,
+  int Function(dynamic currentValue, dynamic nextValue) calculateChangedBits,
```

Comment:
Starting up this conversation again; could we make this generic? Could something like this work?
```dart
ReactDartContext createContext<TValue>([
  TValue defaultValue,
  int Function(TValue currentValue, TValue nextValue) calculateChangedBits,
]) {
```
You mentioned running into issues with generics before; if you're still getting those, could you provide more details around what's not working?